jailbreaks.json

```diff
@@ -75,6 +76,22 @@
         "Linux"
       ]
     },
+    {
+      "jailbroken": true,
+      "name": "Extra_Recipe+YaluX (beta, unstable)",
+      "version": "beta4",
+      "url": "http://https://yalu.qwertyoruiop.com/",
```

Comment:
you've specified the protocol of the URL twice here

Reply:
Oh lmao. I didn't see that. Sorry.